`TestChoiceReplaceFalse` fails on Windows

```
05:36:09.318052 STDOUT 4968]	=========================== short test summary info ===========================	
05:36:09.318052 STDOUT 4968]	FAILED cupy_tests\random_tests\test_generator.py::TestChoiceReplaceFalse_param_0_{a=5, size=2}::test_dtype_shape - AssertionError: assert dtype('int64') == 'long'	
05:36:09.318052 STDOUT 4968]	 +  where dtype('int64') = array([0, 4]).dtype	
05:36:09.318052 STDOUT 4968]	FAILED cupy_tests\random_tests\test_generator.py::TestChoiceReplaceFalse_param_1_{a=5, size=(2, 2)}::test_dtype_shape - AssertionError: assert dtype('int64') == 'long'	
05:36:09.318052 STDOUT 4968]	 +  where dtype('int64') = array([[0, 4],\n       [3, 1]]).dtype	
05:36:09.318747 STDOUT 4968]	FAILED cupy_tests\random_tests\test_generator.py::TestChoiceReplaceFalse_param_2_{a=5, size=()}::test_dtype_shape - AssertionError: assert dtype('int64') == 'long'	
05:36:09.318780 STDOUT 4968]	 +  where dtype('int64') = array(0).dtype	
05:36:09.318780 STDOUT 4968]	= 3 failed, 108982 passed, 3995 skipped, 775 deselected, 598 xfailed, 2613 warnings in 14648.42s (4:04:08) =	
05:36:09.318780 STDOUT 4968]	------------------------------------------------------------------------------------------
```
https://ci.preferred.jp/cupy.win.cuda129/207386/

--- a/tests/cupy_tests/random_tests/test_generator.py
+++ b/tests/cupy_tests/random_tests/test_generator.py
@@ -902,9 +902,9 @@ class TestChoiceReplaceFalse(RandomGeneratorTestCase):
             expected_shape = self.size
         if isinstance(self.a, numpy.ndarray):
             expected_dtype = 'float'
-            assert v.dtype == expected_dtype
         else:
-            assert v.dtype == numpy.dtype('l') or v.dtype == numpy.int64
+            expected_dtype = 'int64'
+        assert v.dtype == expected_dtype
         assert v.shape == expected_shape
 
     @_condition.repeat(3, 10)

